registry/storage/shares.go

```diff
@@ -210,48 +210,57 @@ func (s *sharesStorage) Save(rw basedb.ReadWriter, shares ...*types.SSVShare) er
 		}
 	}
 
-	s.mu.Lock()
-	defer s.mu.Unlock()
+	s.logger.Debug("save validators to shares storage", zap.Int("count", len(shares)))
```

Comment:
nit: `save validators to shares storage` -> `save validator shares to shares storage`

Reply:
oops, missed this one 😓 

we can probably do this in next PR to storage